File: codigos algoritmicos javascript/ejercicio23.js
console.log("Calcular salario total con horas extra");
salario =1500000;
horasExtra = 15;

if (salario > 0) {

    if (horasExtra >= 0) {

        let valorHora = salario / 240;
        let valorHoraExtra = valorHora * 1.5;

        let pagoExtra = horasExtra * valorHoraExtra;

        let salarioTotal = salario + pagoExtra;

        console.log("Salario base:", salario);
        console.log("Pago extra:", pagoExtra);
        console.log("Salario total:", salarioTotal);

    } else {
        console.log("Las horas extra no pueden ser negativas");
    }

} else {
    console.log("El salario debe ser mayor a 0");
}

File: codigos algoritmicos python/ejercicio23.py

salario = float(input("Ingrese salario base: "))
horas_extra = int(input("Ingrese horas extra: "))

if salario > 0:
    
    if horas_extra >= 0:
        
        valor_hora = salario / 240
        valor_hora_extra = valor_hora * 1.5
        
        pago_extra = horas_extra * valor_hora_extra
        
        salario_total = salario + pago_extra
        
        print("Salario base:", salario)
        print("Pago extra:", pago_extra)
        print("Salario total:", salario_total)
    
    else:
        print("Las horas extra no pueden ser negativas")

else:
    print("El salario debe ser mayor a 0")

